[DataZoom] new integration (#1262)
DataDog/integrations-extras: datazoom/README.md

> # Agent Check: datazoom

Reviewer: ```suggestion
# Agent Check: Datazoom
```

## Overview

> The [Datazoom Datadog Connector][1] sends Collector data to a configured Datadog destination, available for query in Log Explorer.

Reviewer: ```suggestion
Datazoom is a video data platform that gathers data from endpoints through an ecosystem of collectors.

The [Datazoom Datadog Connector][1] sends collector data to Datadog, where you can query the data in your [Log Explorer][3].

Datazoom sends data set at the INFO level.
```

## Setup

### Installation

> The Datazoom integration emits logs to Datadog.

Reviewer: ```suggestion
The Datazoom integration emits logs to Datadog. No installation is required on the Datadog side.
```

### Configuration

> - Visit Datazoom's integation [documentation][1].

Reviewer: ```suggestion
- Visit Datazoom's integation [documentation][1] for details on how to configure the Datazoom Datadog Connector.
```

### Dashboard

> Datazoom logs dashboard is available [here](../dashboard/lists/preset/3?q=datazoom). 

Reviewer: ```suggestion
See the [Datazoom logs dashboard][4].
```

## Data Collected

### Metrics

Datazoom does not include any metrics.

### Service Checks

Datazoom does not include any service checks.

### Events

Datazoom does not include any events.

## Troubleshooting

Need help? Contact [Datadog support][2].

[1]: https://help.datazoom.io/hc/en-us/articles/360042494512-Datadog

> [2]: https://docs.datadoghq.com/help/

Reviewer: ```suggestion
[2]: https://docs.datadoghq.com/help/
[3]: https://app.datadoghq.com/logs
[4]: https://app.datadoghq.com/dashboard/lists/preset/3?q=datazoom
```
